README.md

```diff
@@ -2,3 +2,19 @@
 Reach plugin for Magento v2.2 & v2.3.
 
 If you are interested in using this plugin please contact sales@withreach.com
+
+# Deployment Pipeline
+1. work on dev feature branch
+2. open PR, merging to QA branch
+3. PR approved
+4. merge to QA branch
+5. delete feature branch
+6. deploy to QA environment
+   1. Access QA EC2 Server
```

Comment:
Considering this is a public repo do we want to expose that were using a EC2 server?

Reply:
well, they can see everything including this PR haha and you're right. Updated.

I wonder if we can update it so that only certain branches are public?